code/modules/client/preference_setup/general/04_equipment.dm

```diff
@@ -212,6 +212,6 @@
 			set_backpack_metadata(bo, bt, new_metadata)
 			return TOPIC_REFRESH_UPDATE_PREVIEW
 	else if(href_list["change_cash_choice"])
-		pref.starting_cash_choice = next_in_list(pref.starting_cash_choice, typesof(/decl/starting_cash_choice))
+		pref.starting_cash_choice = next_in_list(pref.starting_cash_choice, decls_repository.get_decls_of_type(/decl/starting_cash_choice))
```

Comment:
Should this be get_decl_paths_of_type? I don't see any changes that make starting_cash_choice take a decl rather than a typepath.